backend/src/test/java/wooteco/retrospective/dao/member/MemberDaoTest.java

```diff
@@ -0,0 +1,76 @@
+package wooteco.retrospective.dao.member;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import wooteco.retrospective.domain.member.Member;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@JdbcTest
+class MemberDaoTest {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+    private MemberDao memberDao;
+    private Member memberPika;
+    private Member memberLion;
```

Comment:
```suggestion
    @Autowired
    private JdbcTemplate jdbcTemplate;
    
    private MemberDao memberDao;
    private Member memberPika;
    private Member memberLion;
```
DI받는 객체와 내부 선언되는 필드는 구분해주세요